src/diffusers/models/unet_2d_condition.py

```diff
@@ -634,6 +653,9 @@ def forward(
             else:
                 emb = emb + class_emb
 
+        if self.time_embed_act is not None:
+            emb = self.time_embed_act(emb)
+
```

Comment:
optional activation of time embeddings once at at the beginning of the unet

Reply:
Out of curiosity. 

Is it being used in the private fork?

Reply:
yessir!